rts/Rendering/GlobalRendering.h

```diff
@@ -346,6 +346,10 @@ class CGlobalRendering {
 	int glslMaxStorageBufferBindings;
 	int glslMaxStorageBufferSize; ///< in bytes
 
+	std::array<int, 3> csMaxInvocations;
```

Comment:
Why 3?

Reply:
XYZ dimensions

Reply:
consider some sort of
```suggestion
	std::array<int, 3> csMaxInvocations; // XYZ dimensions
```
or
```suggestion
	std::array<int, 3> csMaxInvocationsXYZ;
```
or
```suggestion
	std::array<int, std::size(std::declval<float3>().xyz)> csMaxInvocations;
```